.github/actions/docker-build-and-push/action.yaml

```
@@ -70,133 +67,130 @@ runs:
           }
         skip-extraction: true
 
-    - name: Set Docker tags
-      id: set-docker-tags
-      run: |
-        tags=()
-        if [ "${{ github.event_name }}" == "push" ] && [ "${{ github.ref_type }}" == "tag" ]; then
-            tags+=("$(echo "${{ github.ref }}" | sed -E 's/.*([vV][0-9]+\.[0-9]+\.[0-9]+).*/\1/')")
-        fi
-
-        tags+=("{{date 'YYYYMMDD'}}")
-        tags+=("${{ inputs.tag-prefix }}")
-
-        # Output multiline strings: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings
-        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
-        echo "tags<<$EOF" >> $GITHUB_OUTPUT
-        echo "$(printf "%s\n" "${tags[@]}")" >> $GITHUB_OUTPUT
-        echo "$EOF" >> $GITHUB_OUTPUT
+    - name: Get current date
+      id: date
+      run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
       shell: bash
 
     - name: Docker meta for autoware:base
       id: meta-base
       uses: docker/metadata-action@v5
       with:
         images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
-        tags: ${{ steps.set-docker-tags.outputs.tags }}
+        tags: |
+          type=raw,value=base${{ inputs.tag-suffix && '-' }}${{ inputs.tag-suffix }}
+          type=raw,value=base${{ inputs.tag-suffix && '-' }}${{ inputs.tag-suffix }}-${{ steps.date.outputs.date }}
         bake-target: docker-metadata-action-base
         flavor: |
           latest=false
-          suffix=-base${{ inputs.tag-suffix }}
 
     - name: Docker meta for autoware:core-devel
       id: meta-core-devel
       uses: docker/metadata-action@v5
       with:
         images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
-        tags: ${{ steps.set-docker-tags.outputs.tags }}
+        tags: |
+          type=raw,value=core-devel${{ inputs.tag-suffix && '-' }}${{ inputs.tag-suffix }}
```

I think it is better for the readability but I'm not sure it is valid for the GitHub Action's syntax. 
@oguzkaganozt  Could you test it with this modification?

```suggestion
          type=raw,value=core-devel${{ (inputs.tag-suffix && '-') + inputs.tag-suffix }}
```

---

Added the suggestion new workflow run -> https://github.com/autowarefoundation/autoware/actions/runs/10881207111

---

[The workflow fails with an error](https://github.com/autowarefoundation/autoware/actions/runs/10881207111/job/30189674281#step:6:2) so we have to stick with the current implementation. @youtalk ?

---

Yes, exactly. I'm sorry but please revert the commit.

---

I made the revert commit by my self.
https://github.com/autowarefoundation/autoware/pull/5216/commits/48ff41ca52a8e61f0e1ca97e759378bcbfa5b6bb